src/electron/electron-builder.json

```diff
@@ -20,7 +20,8 @@
     ],
     "files": ["third_party/outline-go-tun2socks/win32"],
     "icon": "build/icons/win/icon.ico",
-    "sign": "src/electron/windows/electron_builder_signing_plugin.cjs"
+    "sign": "src/electron/windows/electron_builder_signing_plugin.cjs",
+    "signingHashAlgorithms": ["sha256"]
```

Comment:
this will immediately disable the support of Windows 7 (pre SP1), even with the USB signing cert. I'll let @fortuna / @maddyhof to decide.

Reply:
Sounds good!